environment.yml

```diff
@@ -106,6 +106,7 @@ dependencies:
     - example
     - Image
     - sphinx_autobuild
+    - sphinx_copybutton
```

Comment:
The installation name for this extension is `sphinx-copybutton` and already exists in the list.
See [here](https://sphinx-copybutton.readthedocs.io/en/latest/index.html)

Reply:
Sorry I overlooked that. I will remove the duplicate and order the sphinx dependencies alphabetically.